src/components/footer/FooterComponent.tsx

```diff
@@ -0,0 +1,149 @@
+export default function FooterComponent() {
+    return (
+        <div>
+            <footer className="text-center lg:text-left bg-gray-100 text-gray-600 mt-3">
+                <div className="flex justify-center items-center lg:justify-between p-6 border-b border-gray-300">
+                    <div className="mr-12 hidden lg:block">
+                        <span>Get connected with us on social networks:</span>
+                    </div>
+                    <div className="flex justify-center">
+                        <a href="#!" className="mr-6 text-gray-600">
+                            <svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="facebook-f" className="w-2.5" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512">
```

Comment:
You should make this a file `.svg` in `src/assets/images` and create an object or an arr to store this list of SVG. After that, you can render it with the loop.